lua/neogit/popups/tag/actions.lua

```diff
@@ -0,0 +1,161 @@
+local M = {}
+local git = require("neogit.lib.git")
+local client = require("neogit.client")
+local utils = require("neogit.lib.util")
+local FuzzyFinderBuffer = require("neogit.buffers.fuzzy_finder")
+
+--- Gets revisions
+---@return table
+local function get_revisions()
+  local revisions = git.cli["for-each-ref"].format('"%(refname:short)"').call():trim().stdout
+  for i, str in ipairs(revisions) do
+    revisions[i] = string.sub(str, 2, -2)
+  end
+  return revisions
+end
```

Comment:
I think this could get extracted to `git.refs.list()`

Reply:
Is there someway to do that in the config tables in the `neogit/lib/git/cli.lua` 🤔

Reply:
Hmm... nah, don't think so. It'll match `log.list()` and `remote.list()` anyways, so I think its a good fit in a lib.